public/media-hunt/generaldata.js

```diff
@@ -0,0 +1,151 @@
+const normalization = (data) => ({
+  title: data.original_name || data.original_title,
+  year: data.first_air_date || data.release_date,
+  description: data.overview,
+  image: data.poster_path,
+  review: data.vote_average,
+});
+
+const nameCharacterResponse = {
+  cast: [
+    {
+      adult: false,
+      gender: 2,
+      id: 1372,
+      known_for_department: "Acting",
+      name: "Karl Urban",
+      original_name: "Karl Urban",
+      popularity: 35.257,
+      profile_path: "/bsAnEFgVm5kn8DbBZKfnlLNll89.jpg",
+      character: "Billy Butcher",
+      credit_id: "5baaac490e0a2664d400bb5a",
+      order: 0,
+    },
+    {
+      adult: false,
+      gender: 2,
+      id: 1030513,
+      known_for_department: "Acting",
+      name: "Jack Quaid",
+      original_name: "Jack Quaid",
+      popularity: 26.155,
+      profile_path: "/1BRiWg8oMkxqxH4YfEqpZCbWfFC.jpg",
+      character: "Hughie Campbell",
+      credit_id: "5d4821012d1e405083c02531",
+      order: 1,
+    },
+  ],
+  id: 76479,
+};
+
+const actor = nameCharacterResponse.cast.map((actorAndCharacter) => {
+  glumac = {
+    name: actorAndCharacter.name,
+    character: actorAndCharacter.character,
+  };
+  return glumac;
+});
+
+const printStuffToScreen = (normalization, nameCharacterResponse) => {
+  console.log([
+    normalization.title,
+    normalization.year,
+    normalization.description,
+    normalization.image,
+    normalization.review + " " + showStars(normalization.review / 2),
+    nameCharacterResponse,
+  ]);
+};
+
+function showStars(numberOfStars) {
+  if (numberOfStars >= 1 && numberOfStars <= 1.4) return "✡️";
+  if (numberOfStars > 1.4 && numberOfStars <= 1.9) return "✡️ ⭐";
+  if (numberOfStars > 1.9 && numberOfStars <= 2.4) return "✡️ ✡️";
+  if (numberOfStars > 2.4 && numberOfStars <= 2.9) return "✡️ ✡️ ⭐";
+  if (numberOfStars > 2.9 && numberOfStars <= 3.4) return "✡️ ✡️ ✡️";
+  if (numberOfStars > 3.4 && numberOfStars <= 3.9) return "✡️ ✡️ ✡️ ⭐";
+  if (numberOfStars > 3.9 && numberOfStars <= 4.4) return "✡️ ✡️ ✡️ ✡️";
+  if (numberOfStars > 4.4 && numberOfStars < 5) return "✡️ ✡️ ✡️ ✡️ ⭐";
+  if (numberOfStars === 5) return "✡️ ✡️ ✡️ ✡️ ✡️";
```

Comment:
It goes to 10.
Also, can we have a more programatic approach for this, not a hard-coded one?

Reply:
new functionality created